core/src/main/java/org/apache/accumulo/core/file/rfile/PrintInfo.java

```diff
@@ -165,9 +166,8 @@ public void execute(final String[] args) throws Exception {
       System.out
           .println("Reading file: " + path.makeQualified(fs.getUri(), fs.getWorkingDirectory()));
 
-      AccumuloConfiguration aconf = SiteConfiguration.getInstance();
       CachableBlockFile.Reader _rdr = new CachableBlockFile.Reader(fs, path, conf, null, null,
-          aconf, CryptoServiceFactory.getConfigured(aconf));
+          siteConfig, CryptoServiceFactory.getConfigured(siteConfig));
```

Comment:
This shouldn't have ever been using the site configuration. It should use the system configuration instead, because some params might be set in ZooKeeper.

Reply:
It's not possible to use the system configuration from core. This class would need to be moved to a package in server-base like `org.apache.accumulo.server.util`